### PR TITLE
Graduate privateHostedZoneAWS

### PR DESCRIPTION
**Pull request by patrickdillon**
Shared VPC with a private hosted zone from another account has been verified by QE & ROSA.

Periodic is being added in https://github.com/openshift/release/pull/40670.

### CODE REVIEW

**Comment by mrunalp** [comment]
@deads2k @JoelSpeed ptal

**Comment by patrickdillon** [comment]
Will fix the verify test tomorrow-it doesn't like that I deleted the techpreview test file. Wasn't sure if I should leave it empty (or barebones) or delete it. Looks like the former.

**Review by JoelSpeed — COMMENTED** [review]
Yep, need to not remove those tests. We will leave the techpreview/custom resources in place for now

You'll need a PR to cluster-config-operator to prove this promotion to stable doesn't break anything

File: config/v1/stable.dns.testsuite.yaml [hunk 1]
```diff
@@ -12,3 +12,103 @@ tests:
...
       spec: {}
+  - name: Should be able to create a minimal DNS
+    initial: |
+      apiVersion: config.openshift.io/v1
+      kind: DNS
+      spec: {} # No spec is required for a DNS
+    expected: |
+      apiVersion: config.openshift.io/v1
+      kind: DNS
+      spec: {}
```
[the remark this answers is not in the document]
Comment (patrickdillon) [reply config/v1/stable.dns.testsuite.yaml]: Yes! Copypasta fixed.

**Comment by patrickdillon** [comment]
> Yep, need to not remove those tests. We will leave the techpreview/custom resources in place for now

Fixed.

> You'll need a PR to cluster-config-operator to prove this promotion to stable doesn't break anything

https://github.com/openshift/cluster-config-operator/pull/328

Thanks!

**Pull request merged**